Library/Homebrew/cask/lib/hbc/artifact/moved.rb

```diff
@@ -62,14 +62,16 @@ def move_back(skip: false, force: false, command: nil, **options)
           raise CaskError, "It seems the #{self.class.english_name} source '#{target}' is not there."
         end
 
-        ohai "Moving #{self.class.english_name} '#{target.basename}' back to '#{source}'."
+        ohai "Copying #{self.class.english_name} '#{target.basename}' back to '#{source}'."
```

Comment:
Maybe change this to “Backing … up to …”.

Reply:
Updated.